lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -455,18 +455,6 @@ def test_courseware_page_access_without_passing_entrance_exam(self):
                                })
         self.assertRedirects(response, expected_url, status_code=302, target_status_code=200)
 
-    def test_courseinfo_page_access_without_passing_entrance_exam(self):
```

Comment:
While we have the flag, course info still exists.  It seems like this should override the waffle flag, rather than being deleted.

Reply:
changed